packages/components/src/components/Cell/NavigationCell.tsx

```diff
@@ -29,6 +29,12 @@ export type NavigationCellProps = {
      * Name of the icon to use as a left adornment.
      */
     iconName?: IconName;
+    /**
+     * Additional titles to be shown in the cell.
+     */
+    additionalTitles?: {
+        title: string;
```

Comment:
Since the prop name is `additionalTitles`, maybe we dont need each element in the list to be an object with a single `title` field. What do you think about just having a list of strings?